extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -284,6 +284,15 @@ class AmpApesterMedia extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    const elementHeight = this.element.getAttribute('height');
+    const extraHeight =
+      this.element.getAttribute('data-apester-extra-height') || 0;
+    const extraMargin = extraHeight > 0 ? 20 : 0;
+    setStyles(this.element, {
+      'height': px(
+        parseInt(elementHeight, 10) + parseInt(extraHeight, 10) + extraMargin
```

Comment:
Why isn't this just sized appropriately to begin with? That would reduce the possibility of layout shift.

Reply:
Unit (apester media block) is loaded first and then loaded video ad block, it cause shift, we reserved height with margin before video ad block will be inserted.